includes/wp-admin/class-welcome-fields.php

```diff
@@ -258,7 +258,7 @@ public static function render_step_fediverse_intro() {
 			</div>
 			<div class="step-content">
 				<div class="step-text">
-					<h3><?php \esc_html_e( 'Watch a short video: What&#8217;s the Fediverse?', 'activitypub' ); ?></h3>
+					<h3><?php \esc_html_e( 'Watch a short video: "What is the Fediverse?"', 'activitypub' ); ?></h3>
```

Comment:
I've been wondering if we need to mention the video specifically at all, or whether this could be more general, like getting an intro to the Fediverse. Maybe something like "New to the Fediverse? Start Here"

Reply:
also fine!